test/parallel/test-runner-mock-timers.js

```diff
@@ -844,4 +844,38 @@ describe('Mock Timers Test Suite', () => {
       clearTimeout(id);
     });
   });
+
+  describe('Api should have same public properties as original', () => {
```

Comment:
What about adding tests that go over all properties so we won't have that problem again?

Reply:
Do you meanb internal mock properties  that are not part of the original Timeout object?